### PR TITLE
1.19.2 doc updates

### PR DESCRIPTION
**Pull request by keirthana**
Contains:
* Versions changes
* Release dates and roadmap updates
* Document `--force` option for appliance upgrade
* Placeholder content for 1.19.2 component versions - this needs to be updated closer to the release.

### CODE REVIEW

File: ref/roadmap.md [hunk 1]
```diff
@@ -11,5 +11,6 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 ## Roadmap
 | Release | Target Date | What to expect| 
 |---------|-------------|---------------|
-| 1.19.2 | October 11 2023 | * Android security updates for October 2023<br/>* Bug fixes |
 | 1.20.0 | November 15 2023 | * Support for running Anbox Cloud on VMs<br/>* Lazy image download<br/>* Support for Android 14<br/>* Vulkan support for NVIDIA GPUs<br/>* Enhancements to the streaming experience|
```
Comment (ajanon) [inline ref/roadmap.md:14]: We should remove Android 14 support from that list.

**Review by ajanon — CHANGES_REQUESTED** [review]
One comment.

**Review by ajanon — APPROVED** [review]
LGTM

**Review by jat-canonical — APPROVED** [review]
LGTM

**Pull request merged**